tests/ctst/world/Zenko.ts

```diff
@@ -280,6 +288,78 @@ export default class Zenko extends World {
         return result.data.access_token;
     }
 
+    /**
+     * Creates an assumed role session with a duration of 12 hours.
+     * @Param {boolean} crossAccount - If true, the role will be assumed cross account.
+     * @returns {undefined}
+     * @param crossAccount
+     */
+    async prepareAssumeRole(crossAccount: boolean = false) {
+        // Getting account ID
+        const account = await SuperAdmin.getAccount({
+            name: this.parameters.AccountName || Constants.ACCOUNT_NAME,
+        });
+        Zenko.additionalAccountsCredentials[account.name] = {
+            AccessKey: CacheHelper.parameters.AccessKey,
+            SecretKey: CacheHelper.parameters.SecretKey,
+        };
```

Comment:
It might be a good idea to save the current account in the local list in Zenko init functions, so that we can use any account ing a generic way across all tests, what do you think?